package.json

```diff
@@ -6,16 +6,38 @@
     "dev": "vite",
     "generate": "openapi-generator-cli generate",
     "build": "vue-tsc && vite build",
+    "lint": "stylelint '**/*.{vue,scss}' && prettier . --check && eslint --ext ts,js,vue .",
     "preview": "vite preview"
   },
   "dependencies": {
-    "vue": "^3.2.45"
+    "vue": "^3.2.45",
+    "vue-router": "^4.1.6"
   },
   "devDependencies": {
     "@openapitools/openapi-generator-cli": "^2.5.2",
+    "@types/eslint": "^8.21.1",
+    "@types/node": "^18.14.0",
+    "@typescript-eslint/eslint-plugin": "^5.52.0",
+    "@typescript-eslint/parser": "^5.52.0",
     "@vitejs/plugin-vue": "^4.0.0",
+    "eslint": "^8.34.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-import-resolver-alias": "^1.1.2",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-promise": "^6.1.1",
```

Comment:
At work we use [`"eslint-plugin-prettier"`](https://www.npmjs.com/package/eslint-plugin-prettier) to integrate ESLint and Prettier. I think correctly set-up it disables all (potentially conflicting) ESLint formatting rules, and instead enforces that the code is formatted the Prettier way.

Reply:
You're right in that `eslint-plugin-prettier` integrates ESLint with Prettier, but it doesn't change any rules - that's what `eslint-config-prettier` is for.

I chose to separate Prettier and ESLint (see 5a047e35bd9318e6e992f002242af478d05c9aa5)